lib/jwt_signed_request/middlewares/faraday.rb

```diff
@@ -16,7 +16,7 @@ def call(env)
           method:     env[:method],
           path:       env[:url].request_uri,
           headers:    env[:request_headers],
-          body:       env.fetch(:body, ::JWTSignedRequest::EMPTY_BODY),
+          body:       env[:body] || (env[:body] = ::JWTSignedRequest::EMPTY_BODY),
```

Comment:
Would it be less surprising (more readable) to set this beforehand? eg:

```ruby
env[:body] ||= ::JWTSignedRequest::EMPTY_BODY
@jwt_token = ::JWTSignedRequest.sign(

```

So this would become simply:

```suggestion
          body:       env[:body],
```

Great catch btw!